pgtrigger/core.py

```diff
@@ -187,9 +190,9 @@ class Timing(_Primitive):
 class Condition:
     """For specifying free-form SQL in the condition of a trigger."""
 
-    sql: str = None
+    sql: str | None = None
```

Comment:
I'm assuming you'll need to use `Union` for 3.8 compat, right?

Reply:
Nope, since I added `from __future__ import annotations`

Reply:
You can use it in earlier versions when type annotations are stringified.

Reply:
Oh nice, I should convert this elsewhere. I do believe there is a py3.8 bug if you check circle though

```    from pgtrigger.runtime import (
  File "/code/pgtrigger/runtime.py", line 154, in <module>
    def ignore(*uris: str, databases: Union[List[str], None] = None) -> Generator[None]:
TypeError: 'ABCMeta' object is not subscriptable```

Reply:
Ah, had to add a future imports for that.